### ISSUE TITLE
Improve WCAG compliance using WAVE tool across RentARoom

### ISSUE THREAD

**Issue by jmccann897** [jmccann897/RentARoom#56]
| Page| Number of Errors Reported Using WAVE Tool | Image
|-|-|-|
| `../User/Home/SearchResults` | 4 (Missing form labels) | ![Image](https://github.com/user-attachments/assets/b967d1f5-797a-4382-8e5b-22b62a3f9c32) 
| `../User/Home/Details/{propertyId}` | 3 (Empty buttons + empty form label) | ![Image](https://github.com/user-attachments/assets/4f1e90a1-560b-48e3-bf72-6635823d90e5)
| `../User/Map/MapSearch` | 4 (Missing form labels) | ![Image](https://github.com/user-attachments/assets/9fe7608f-e033-46f4-8555-768070c99925)
| `../User/Notification/Chat`| 2 (Missing form label and Empty form label)  | ![Image](https://github.com/user-attachments/assets/c2b9f8c9-03b8-4c37-833e-6ad20d1387d2)
| `../Agent/Properties` | 1 (Empty table header) | ![Image](https://github.com/user-attachments/assets/dcf61048-a161-42dd-959f-e328bcc781de)
| `../Agent/Properties/Upsert`  | 4 (Missing form labels)  | ![Image](https://github.com/user-attachments/assets/4eb60caa-d0da-4df9-a7d2-aa1e6159d21f)
| `../Agent/Properties/Upsert/?id={propertyId}` | 10 (Missing alternative text, Missing form label and Empty buttons)   | ![Image](https://github.com/user-attachments/assets/d67c2326-9dc6-410a-a505-17cbde08ad2b)
